cmd/nginxproxy/cont-init/01-nginx-pid-dir

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/execlineb -P
+
+s6-mkdir -p /run/nginx/
```

Comment:
This is necessary because the nginx package in Alpine 3.4 is configured differently, and depends on an init script with openrc to create the pidfile directory. This is a best practice if you're using openrc, but I don't really like openrc.